doc/user/content/sql/begin.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "BEGIN"
+menu:
+  main:
+    parent: "sql"
+---
+
+`BEGIN` starts a transaction block.
+
+## Syntax
+
+{{< diagram "begin.svg" >}}
+
+Supported `transaction_mode` option values:
+
+Value | Description
+------|----------
+`ISOLATION LEVEL SERIALIZABLE` | A no-op because transactions are always serializable.
+
+## Details
+
+`BEGIN` starts a transaction block.
+All statements in a transaction block will be executed in a single transaction until an explicit `COMMIT` or `ROLLBACK` is given.
+
+Transactions in Materialize do not support interleaving arbitrary kinds of statements, but instead are either **read only** or **write only**, determined by the first statement after the `BEGIN`.
+
+A **read-only** transaction starts with a [`SELECT`](/sql/select) statement and allows only `SELECT` statements.
+Because Materialize does not know which objects (sources, tables, or views) will be queried during the transaction, the objects in the first `SELECT` and any other object in the same schemas are assumed to be possible query targets.
+Other queries can only reference these objects.
```

Comment:
Does this mean that you can only reference objects referenced in the first `SELECT` statement?

Reply:
only objects adjacent to those in the first query. added "same-schema objects" to attempt to clarify.